src/Nethermind/Nethermind.AuRa.Test/Contract/TxPriorityContractTests.cs

```diff
@@ -111,6 +111,8 @@ public async Task priority_should_return_correctly()
         }
         
         [Test]
+        [Retry(3)]
+        [Explicit]
```

Comment:
explicit will prevent the test from running

Reply:
> explicit will prevent the test from running

I know, but that is how we marked the rest of the tests in that class. Removed explicit.